src/main/java/gregtech/api/capability/impl/MultiblockFuelRecipeLogic.java

```diff
@@ -99,4 +103,19 @@ public void invalidate() {
         super.invalidate();
         totalContinuousRunningTime = 0;
     }
+
+    public String[] getRecipeFluidInputAmount() {
+        IRotorHolder rotorHolder = metaTileEntity instanceof MetaTileEntityLargeTurbine ? ((MetaTileEntityLargeTurbine) metaTileEntity).getRotorHolder() : null;
```

Comment:
This should check if the mte is instanceof MultiblockWithDisplayBase, and then fetch the rotor holder from the multiblock's abilities list. That way it can catch all possible multiblocks with rotor holders, such as those registered by addons

Reply:
fixed with cf6ce20